Imagekit/Helper/TransformationTypes.cs

```diff
@@ -407,6 +218,18 @@ public Transformation EffectGray()
         return this.Add("e-grayscale", "true");
     }
 
+    /// <summary></summary>
+    public Transformation EffectShadow()
+    {
+        return this.Add("e-shadow", "true");
```

Comment:
There is no such transformation as e-shadow-true, e-shadow has a syntax like this e-shadow-<blur>_<saturation>_<x-offset>_<y-offset> or just e-shadow 
https://imagekit.io/docs/effects-and-enhancements#shadow---e-shadow

Reply:
fixed it